### PR TITLE
Portfolio mobile view

### PR DESCRIPTION
**Pull request by JackMwisa**
**styling my portfolio site for mobile view**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -103,11 +110,89 @@ header .menu {
...
+
+.menu-content .menu-list-item {
+  margin-top: 2.5rem;
+  padding: 0 1.75rem;
+}
+
+.menu-content a {
+  font-size: 2rem;
+  line-height: 4rem;
+  font-weight: 600;
```
Comment (vechicin) [inline css/styles.css:146]: None of your `font-weight` rules are working. That is most likely caused by not importing the right weights from Google Fonts. Please refer to [this video](https://www.youtube.com/watch?v=mRoBBOTYNz4&t=184s) so you can fix this issue.

Screen Shot 2022-07-08 at 7 45 46

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @JackMwisa ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (JackMwisa) [reply css/styles.css:146]: @vechicin thank you for this video, I have imported the font in all it format and applied the suggested changes

**Comment by JackMwisa** [comment]
@vechicin can you please check my project, I have added the changes you suggested.

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @JackMwisa  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- Design looks great 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**